cpp/autosar/src/rules/A13-3-1/FunctionThatContainsForwardingReferenceAsItsArgumentOverloaded.ql

```diff
@@ -26,5 +26,7 @@ where
   not isExcluded(f,
     OperatorsPackage::functionThatContainsForwardingReferenceAsItsArgumentOverloadedQuery()) and
   not f.isDeleted() and
-  f = c.getAnOverload()
+  f = c.getAnOverload() and
+  //allow for overloading with different number of parameters
```

Comment:
```suggestion
  // allow for overloading with different number of parameters, because there is no
  // confusion on what function will be called.
```

Adding reason for exception not described in the standard.